features/admin.extensions.v1/configs/common.tsx

```diff
@@ -61,6 +63,35 @@ export const commonConfig: CommonConfig = {
                         path: AppConstants.getPaths().get("USERSTORE_TEMPLATES"),
                         protected: true,
                         showOnSidePanel: false
+                    },
+                    {
```

Comment:
Let's evaluate the possibility of moving these to https://github.com/wso2/identity-apps/blob/master/apps/console/src/configs/routes.tsx
And removing from extension configs.

Reply:
There's one route that needs to be replaced in extensions. Hence keeping this as it is.